SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.cc

```diff
@@ -350,9 +350,9 @@ SiPixelDigitizerAlgorithm::SiPixelDigitizerAlgorithm(const edm::ParameterSet& co
 }
 
 std::map<int, SiPixelDigitizerAlgorithm::CalParameters, std::less<int> > SiPixelDigitizerAlgorithm::initCal() const {
-  using std::cerr;
-  using std::cout;
-  using std::endl;
+  //  using std::cerr;
```

Comment:
@carolinecollard , please, remove these lines forever, because std::cout,... allowed only in analyzers and unit tests.